src/pages/patientView/trialMatch/TrialMatchTableUtils.tsx

```diff
@@ -0,0 +1,27 @@
+import { IArmMatch, IClinicalGroupMatch } from "../../../shared/model/MatchMiner";
```

Comment:
this file can be .ts file because it doesn't involve any jsx.   There is a small build performance hit for .tsx files because the tsx compiler needs to search if for markup.  By giving it .ts extension, we save it from having to do that.

Reply:
Thanks for pointing it out. Will change to ts.